packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/xdsl-meetings.routing.js

```diff
@@ -0,0 +1,58 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('telecom.xdsl-meetings', {
+    url: '/xdsl/:serviceName/meetings',
+    views: {
+      'telecomView@telecom': 'packXdslMeetings',
+    },
+    resolve: {
+      serviceName: /* @ngInject */ ($transition$) =>
+        $transition$.params().serviceName,
+      loadMeetings: /* @ngInject */ ($rootScope, OvhApiXdsl, serviceName) => {
```

Comment:
Can't we avoid injecting `$rootScope` ?

Reply:
No I need it, the API could answer "pending" so I need to call again until having an answer (empty or not)